week1/homework/src/index.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
 // Write the homework code in this file
+let http = require('http');
+let port = 8080;
+let state = 10;
+
+
+let server = http.createServer((req, res) => {
+    res.setHeader('Content-type', 'text/html')
+    switch (req.url) {
+        case '/' :
+        break;
+        case '/state':
+            res.write(`<h1> OK :${state}</h1>`);
+            break;
+        case '/add':
+            res.write(`<h1> oK :${state + 1}</h1>`);
+            break;
+        case '/subtract':
+            res.write(`<h1> Ok :${state - 1}</h1>`);
+            break;
+        case '/reset':
+        res.write(`<h1 id = 'state'> oK :${state}</h1>`);
+break;
+default : 
+res.statusCode = 403;
```

Comment:
Not found status code is [404](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404).